<launch>
    <include file="$(find hrl_rfh_summer_2011)/launch/approach_only_setup.launch" />
    <include file="$(find hrl_rfh_fall_2011)/launch/head_registration.launch" />
    <node pkg="hrl_trajectory_playback" name="untuck_l_arm_playback" type="playback.py" 
	  output="screen" respawn="false" 
	  args="--pkl $(find hrl_rfh_fall_2011)/data/untuck_l_arm.pkl --name untuck_l_arm --left" />
    <node pkg="hrl_trajectory_playback" name="untuck_r_arm_playback" type="playback.py" 
	  output="screen" respawn="false" 
	  args="--pkl $(find hrl_rfh_fall_2011)/data/untuck_r_arm.pkl --name untuck_r_arm" />
    <node pkg="hrl_trajectory_playback" name="r_adjust_mirror_playback" type="playback.py" 
	  output="screen" respawn="false" 
	  args="--pkl $(find hrl_rfh_fall_2011)/data/r_adjust_mirror.pkl --name r_adjust_mirror" />
    <node pkg="hrl_rfh_fall_2011" type="ellipsoid_controller.py" name="ellipsoid_controller"
      output="screen" />
</launch>
